### PR TITLE
Refactor variable name for Gamemaster outfit in player protocol

### LINKED ISSUE #4383: Crash when opening outfits window with GM
### Before creating an issue, please ensure:
- [x] This is a bug in the software that resides in this repository, and not a
      support matter (use https://otland.net/forums/support.16/ for support)
- [x] This issue is reproducible without changes to the C++ code in this repository

### Steps to reproduce (include any configuration/script required to reproduce)
1. Login with a GM account.
2. Open customization window or ctrl + r.

### Expected behaviour 
The outfits and mounts window should open normally.

### Actual behaviour 
The client crashes when opening.

### PR DESCRIPTION
**Pull request by omarcopires**
### Pull Request Prelude

- [x] I have followed [proper The Forgotten Server code styling][code].
- [x] I have read and understood the [contribution guidelines][cont] before making this PR.
- [x] I am aware that this PR may be closed if the above-mentioned criteria are not fulfilled.

### Changes Proposed 
Changed Gamemaster outfit creation to use a const static string for outfit name.

[cont]: https://github.com/otland/forgottenserver/wiki/Contributing
[code]: https://github.com/otland/forgottenserver/wiki/TFS-Coding-Style-Guide

### CODE REVIEW

**Comment by ranisalt** [comment]
Why?

**Comment by omarcopires** [comment]
> Why?

The main reason for changing the code in this way is to improve code efficiency and readability.
By introducing a static constant variable with the gamemaster outfit name, we're reducing the number of times the string "Gamemaster" needs to be created and destroyed during program execution. This can be helpful in situations where creating temporary objects can lead to performance bottlenecks.
Additionally, introducing a variable with a descriptive name makes the code more readable and easier to understand. The use of descriptive variables can make the code easier to maintain, especially when multiple parts of the code need to reference the same string.
In summary, the proposed change is a good programming practice that improves code efficiency and readability.

**Comment by ranisalt** [comment]
The literal string already lives in static storage and does not have to be instantiated. Storing in a `static std::string` moves instantiation to runtime, in fact losing performance and wasting more space.

I can't see how `gamemasterOutfitName` is less descriptive than `"Gamemaster"`, but I don't have a strong opinion to block based on that. In that case, `constexpr auto gamemasterOutfitName = "Gamemaster"` is better than a `static std::string`.

(also ChatGPT is too wordy, it writes too much to say too little)
